core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -38,7 +38,14 @@
 .section .data
 .balign 4
 
-#ifdef CFG_BOOT_SYNC_CPU
+/*
+ * For VxWorks Support, the primary core will go through OP-TEE init
+ * without ever touching secondary cores; it is only after the primary
+ * core enters VxWorks, it issues special SMC to request booting of
+ * secondary cores. Thus CPU Sync is not required.
+ */
+
+#if defined (CFG_BOOT_SYNC_CPU) && !defined (CFG_TEE_VXWORKS_SUPPORT)
```

Comment:
You're right it's better solution to modify the "conf.mk" of plat-ls and plat-stm not to force the CPU sync, and rather inject the CFG_BOOT_SYNC_CPU from make command. I prefer the "CFG_BOOT_SYNC_CPU ?= y" solution since it seems more flexible. I'll update the code accordingly.